ocw/lib/gce.py

```diff
@@ -91,13 +91,17 @@ def list_regions(self) -> list:
         return [region["name"] for region in regions]
 
     def list_zones(self, region) -> list:
-        region = (
-            self.compute_client()
-            .regions()
-            .get(project=self.project, region=region)
-            .execute()
-        )
-        return [basename(z) for z in region["zones"]]
+        try:
+            region = (
+                self.compute_client()
+                .regions()
+                .get(project=self.project, region=region)
+                .execute()
+            )
+            return [basename(z) for z in region["zones"]]
+        except (KeyError, HttpError) as exc:
```

Comment:
again the same as in #300  we should not mute such broad exceptions . I assume you trying to handle properly case when google ( very often ) returning regions in list_regions which later in other calls returning "region does not exists " ? generally I agree with this but we must be more precise . what you catching here is too broad ....

Reply:
This is better than the situation we  had before where the exception prevented the cleanup from going on.  We still must catch it, and at least we're doing something with it like logging the error.

Reply:
it is big question to if it is better now . You catching all KeyError/HttpError and only doing debug message about that so they currently FULLY muted in prod instance where we have INFO log level so if something is broken in this call we missing some part of regions from cleanup and NOT getting any notifications about that anywhere I am not sure if it can be called better

Reply:
This was fixed in a subsequent PR which changed log_dbg to log_err.

Reply:
thanks !